Makefile

```diff
@@ -0,0 +1,33 @@
+test:
+	poetry run pytest -v
+
+lint:
+	poetry run pre-commit run -a -v
+
+install:
+	poetry install
+
+migrate:
```

Comment:
The migrations helps to develop the application without interaction with the database, makes sense to put the migrations folder in the .gitignore for now, and keep it to develop only?

Reply:
I understand the purpose of database migrations, the question is, do we have to run one yet?

If we have no reason to run a migration yet, then building this prematurely will make it harder and more confusing for the next developer who has to figure out how to migrate the database for a new release.

Reply:
This might also be a candidate for like a doc/example/migration/ folder or something. Then you have it around to reference, but we're not obliged to test or maintain any migration code prematurely.